kymatio/scattering2d/frontend/base_frontend.py

```diff
@@ -35,6 +35,10 @@ def create_filters(self):
         filters = filter_bank(self.M_padded, self.N_padded, self.J, self.L)
         self.phi, self.psi = filters['phi'], filters['psi']
 
+    def scattering(self, x):
```

Comment:
will this eventually follow the same pattern as `kymatio/scattering1d/frontend/base_frontend.py`?

Reply:
maybe. if needed!